test/parallel/test-buffer-arraybuffer.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const Buffer = require('buffer').Buffer;
+const LENGTH = 16;
+
+var ab = new ArrayBuffer(LENGTH);
+var dv = new DataView(ab);
+var ui = new Uint8Array(ab);
+var buf = new Buffer(ab);
+
+
+assert.ok(buf instanceof Buffer);
+// For backwards compatibility of old .parent property, if the Buffer uses the
```

Comment:
This might just be my ignorance but why should it be undefined? `buffer.parent` isn't even documented so I'm not sure what the contract is here that you're preserving.